.github/workflows/ci.yaml

```diff
@@ -0,0 +1,22 @@
+%YAML 1.1
+---
+name: Run tests
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: ['master', 'production', 'ros2']
+  pull_request:
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/checkout@v2
```

Comment:
Should these be doubled?

Reply:
Yes, the first invocation pulls `ros_buildfarm_config`, and the second pulls `ros_buildfarm`.